carts/views.py

```diff
@@ -27,4 +28,51 @@ def get(self, request):
             "is_checked": False
         } for cart in cart_list]
 
-        return JsonResponse({'MESSAGE': result}, status=200)
\ No newline at end of file
+        return JsonResponse({'MESSAGE': result}, status=200)
+
+    @login_decorator
+    @transaction.atomic()
+    def post(self, request):
+        try:
+            datas           = json.loads(request.body)
+            user            = request.user
+            product_id      = datas["product_id"]
+            products        = datas["product"]
+            target_product  = Product.objects.get(id = product_id)
+            
+            for product in products:
+                quantity    = product["quantity"]
+                graind      = target_product.graindbyproduct_set.get(grainding_id = product["graind"]).grainding
+                size        = target_product.size_set.get(name = product["size"])
+
+                cart, is_bool   = Cart.objects.get_or_create(
+                    user        = user,
+                    product     = target_product,
+                    graind      = graind,
+                    size        = size,
+                    defaults    = {'quantity': quantity}
+                )
+
+                if is_bool == False:
```

Comment:
```suggestion
                if not is_bool:
```
파이썬 PEP 8 프로그래밍 권고에는 비교문 관련해서 어떤 형식을 사용하라고 안내되어 있습니다.

- True/False를 비교할 때는 if문에 변수만 사용하도록 한다.
- 동등성(단순히 값만 비교하는 경우)을 검증할 때는 == 혹은 != 같은 연산자를 사용하도록 한다.
- 동일성(객체의 고유 메모리 주소로 비교하는 경우)을 검증할 때는 is나 is not을 사용한다.

즉 현재의 경우는 True/False만을 비교하기 때문에 `== False`는 붙일 필요가 없습니다.

Reply:
네!! 
수정 하도록 하겠습니다.